app/models/pacbio/library.rb

```diff
@@ -44,7 +44,8 @@ class Library < ApplicationRecord
     validates :primary_aliquot, presence: true, if: -> { pool.blank? }
     accepts_nested_attributes_for :primary_aliquot, allow_destroy: true
 
-    after_create :create_used_aliquot, if: -> { pool.blank? }
+    after_create :create_used_aliquot, :create_tube, if: -> { pool.blank? }
+    before_destroy :check_for_associated_wells, prepend: true
```

Comment:
I know we discussed it this morning but what if it is part of a pool?

Reply:
For this first piece of work none of these 'new libraries' will be in pools. But its a good point and I will add that as an acceptance criteria for the story that involves adding 'new libraries' in pools.